src/ui/map.ts

```diff
@@ -121,15 +144,59 @@ class MapUI {
         }
       }
     };
+
+    const onStyleLoad = async () => {
```

Comment:
NOTE: this `onStyleLoad` hook is to handle custom markers that uses an old pattern that was previously discussed. We've since decided that rather than handling the default marker for them (i.e. defaulting to the custom marker), we're just gonna have them pass in the custom marker id. This change is covered in this follow up PR: https://github.com/radarlabs/radar-sdk-js/pull/153